terraform/services/config/main.tf

```
@@ -0,0 +1,26 @@
+# TODO replace ref with hash after merging
+module "platform" {
+  source    = "github.com/CMSgov/cdap//terraform/modules/platform?ref=plt-1358_sops"
+  providers = { aws = aws, aws.secondary = aws.secondary }
+
+  app         = "cdap"
```

The app here should be bcda for now because we don't have cdap-test and cdap-prod environments set up yet. This should fix the errors we're seeing at https://github.com/CMSgov/cdap/actions/runs/19348268675/job/55354043475?pr=324


```suggestion
  app         = "bcda"
```